examples/next-prisma-starter/.github/workflows/main.yml

```diff
@@ -3,41 +3,50 @@ on: [push]
 jobs:
   e2e:
     env:
-      DATABASE_URL: postgresql://postgres:@localhost:5432/trpcdb
+      # DATABASE_URL: postgresql://postgres:@localhost:5432/trpcdb
       NODE_ENV: test
       NEXTAUTH_SECRET: supersecret
+      PLAYWRIGHT_BROWSERS_PATH: ${{ github.workspace }}/playwright
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        node: ['14.x']
+        node: ['16.x']
         os: [ubuntu-latest]
-    services:
-      postgres:
-        image: postgres:12.1
-        env:
-          POSTGRES_USER: postgres
-          POSTGRES_DB: trpcdb
-        ports:
-          - 5432:5432
+    # services:
+    #   postgres:
+    #     image: postgres:12.1
+    #     env:
+    #       POSTGRES_USER: postgres
+    #       POSTGRES_DB: trpcdb
+    #     ports:
+    #       - 5432:5432
     steps:
       - name: Checkout repo
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - uses: pnpm/action-setup@v2.2.2
+        with:
+          version: 7.12.0
+
+      - name: Use Node ${{ matrix.node }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node }}
+          cache: 'pnpm'
 
-      # - name: Install deps and build (with cache)
-      #   uses: bahmutov/npm-install@v1
-      - run: yarn install
+      - name: Install deps (with cache)
+        run: pnpm install
 
       - name: Next.js cache
-        uses: actions/cache@v2
+        uses: actions/cache@v3
```

Comment:
Be extra careful when editing this -- we won't know if it breaks until it's subtree'd into https://github.com/trpc/examples-next-prisma-starter

Reply:
Tested it here: https://github.com/juliusmarminge/next-prisma-starter/actions